ringtest.py

```diff
@@ -195,6 +204,11 @@ def create_rings():
 
     ## Initialize ##
 
+    if method > 0:
+        h.cvode.condition_order(condition_order)
+        h.cvode.active(1)
+        if (method == 2):
```

Comment:
Nit: parentheses are not necessary.

Reply:
Fixed.  In a separate pull request, it would be a good idea to use black for consistent formatting.